response_test.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"go/token"
+	"reflect"
+	"testing"
+)
+
+func diff(t *testing.T, prefix string, have, want interface{}) {
```

Comment:
im not totally sure what this test is doing. i would create tests that are more readable IE: test response returns 200 get.

Reply:
✅ Test was in the wrong file, moved to the request test file and renamed for easier readability